src/org/traccar/database/PermissionsManager.java

```diff
@@ -235,6 +210,9 @@ public void checkDeviceReadonly(long userId) throws SecurityException {
 
     public void checkUserEnabled(long userId) throws SecurityException {
         User user = getUser(userId);
+        if (user == null) {
+            throw new SecurityException("Unknown account");
```

Comment:
Noticed next thing: if user was logged in and was deleted from server, after refreshing page he will get `NullPointerException`. But now he will get login page.